app/models/miq_report/import_export.rb

```diff
@@ -107,10 +107,21 @@ def import_from_hash(report, options = nil)
     # @param cache [Hash] cache that holds yaml for the views
     def load_from_view_options(db, current_user = nil, options = {}, cache = {})
       filename = MiqReport.view_yaml_filename(db, current_user, options)
-      yaml     = cache[filename] ||= YAML.load_file(filename)
-      view     = MiqReport.new(yaml)
-      view.db  = db if filename.ends_with?("Vm__restricted.yaml")
+      yaml = cache[filename] ||= YAML.load_file(filename)
```

Comment:
@ZitaNemeckova as we discussed I am seeing two usage of load_file method.

So we can use `safe_load` and somehow unify them. Is it possible ?

Reply:
@lpichler Removed the "middle" method and used `load_file` because it was the original one.